Library/Homebrew/cmd/analytics.rb

```diff
@@ -0,0 +1,44 @@
+#:  * `analytics` [`state`]:
+#:    Display analytics state.
+#:
+#:  * `analytics` `on`|`off`:
+#:    Turn on/off Homebrew's analytics.
+#:
+#:  * `analytics` `regenerate-uuid`:
+#:    Regenerate UUID used in Homebrew's analytics.
+
+module Homebrew
+  def analytics
+    config_file = HOMEBREW_REPOSITORY/".git/config"
+
+    case ARGV.named.first
+    when nil, "state"
```

Comment:
I'd like to see every `when` clause of this `case` to be refactorded into a separate method to keep `Homebrew.analytics` short and readable. And I wonder if we should also complain when passing more than one named argument. (Let's try to be more strict in our argument handling for new `brew` commands we introduce.)